package/translate/ja.po

```diff
@@ -85,12 +85,12 @@ msgstr "ブラウザーで編集"
 #: ../contents/ui/AgendaListItem.qml
 msgctxt "agenda date format line 1"
 msgid "MMM d"
-msgstr ""
+msgstr "agenda date format line 1"
```

Comment:
`msgctxt` (message context) is there for the translator to understand the message `msdid`.

In this case you want `msgid "MMM d"` which is used in the date formatting to display `Jun 27`.

`ddd` is `Sat` (short for `Saturday`).

https://doc.qt.io/qt-5/qml-qtqml-qt.html#formatDateTime-method

Reply:
ok I understood